core/psh/psh.c

```diff
@@ -89,7 +89,7 @@ int main(int argc, char **argv)
 	while (write(1, "", 0) < 0)
 		usleep(50000);
 
-	splitname(argv[0], &base, &dir);
+	base = basename(argv[0]);
 
 	/* Run app */
 	if ((app = psh_findapp(base)) != NULL)
```

Comment:
```suggestion
	if ((app = psh_findapp(basename(argv[0]))) != NULL)
```
We can ommit using `base` pointer at all. `basename()` is self explanatory so we do not lose code readability